integration/container/ps_test.go

```diff
@@ -0,0 +1,39 @@
+package container
+
+import (
+	"context"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/integration/util/request"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPsFilter(t *testing.T) {
+	defer setupTest(t)()
+	client := request.NewAPIClient(t)
+	ctx := context.Background()
+
+	_, err := client.ContainerCreate(ctx,
+		&container.Config{
+			Cmd:   []string{"top"},
+			Image: "busybox",
+		},
+		&container.HostConfig{},
+		&network.NetworkingConfig{},
+		"top",
+	)
+	require.NoError(t, err)
+
+	filters := filters.NewArgs()
+	filters.Add("since", "top")
+	filters.Add("before", "top")
+	_, err = client.ContainerList(ctx, types.ContainerListOptions{
+		All:     true,
+		Filters: filters,
+	})
+	require.NoError(t, err)
```

Comment:
If we're going to spend the time on an integration test, I think it should at least test the "success" case where it finds something.

I would say that ideally we could test  this with a unit test, but I understand that is difficult with `Daemon`.

Reply:
Thanks @dnephin. The PR has been updated. Please take a look.